.github/workflows/main.yml

```diff
@@ -21,27 +21,44 @@ jobs:
       TARANTOOL_UPDATE_URL: ${{secrets.TARANTOOL_UPDATE_URL}}
     steps:
       - uses: actions/checkout@v2
+        id: checkout
         with:
+          ref: latest
           submodules: recursive
 
       - name: Fetch submodules
+        id: fetch-submodules
         run: git fetch --recurse-submodules
 
       - run: cmake .
+        id: cmake
       - run: make pull-modules
+        id: make-pull-modules
       - run: make build-modules
+        id: make-build-modules
       - run: make json
+        id: make-json
       - run: make json-ru
+        id: make-json-ru
       - run: make pdf
+        id: make-pdf
       - run: make pdf-ru
+        id: make-pdf-ru
+
+      # set environment variable BRANCH_NAME, will be used in scripts
       - uses: nelonoel/branch-name@v1.0.1
```

Comment:
DEPLOYMENT_NAME (ex. BRANCH_NAME) is a name of the current branch, you can use it instead of another external action